core/workspace_svg.ts

```diff
@@ -2766,7 +2768,33 @@ export class WorkspaceSvg
       return null;
     }
 
-    return this.getBlockById(id) as IFocusableNode;
+    // Search for a specific block.
+    const block = this.getBlockById(id) as IFocusableNode;
```

Comment:
Do you still need this cast?

Reply:
I do not! Good catch--removed in the latest changes.

Reply:
This has been addressed--marking as resolved to keep open only those threads that may need additional discussion.